### ISSUE TITLE
Feature suggestion: Save settings

### ISSUE THREAD

**Issue by Ikabodo** [ChadKeating/rolladvantage-token-stamp-2-foundry#1]
Don' t know if it's possible through the page. But it would be nice to be able to save settings between tokens.

Also. Super nice work!

**Comment by ChadKeating** [comment]
Hey thanks!

This is something I have wanted to do but I am saving for Token Stamp 3. The idea is you'll be able to save local settings including custom borders as well as export/import so you can share settings or back them up.

**Comment by Ikabodo** [comment]
Nice thinking. 
For me, the main feature is speed, and I will definitely use this. Considering I use photoshop for most tokens (since I do fancy masking), but I still need the ability to quickly add temporary tokens mid-sessions.

Are you the developer of the actual site too?

**Comment by ChadKeating** [comment]
Yes I am, and yeah after using it a little within foundry I realise the importance of a presets feature. Token Stamp 2 is like 4/5 years old now so it wasnt really built with integration or presets in mind... and it's a little bloated with the features added over time. I'm hoping to get working on Token Stamp 3 soon.
